packages/cli/src/index.ts

```diff
@@ -19,6 +19,13 @@ signals.forEach((signal) => {
   })
 })
 
+// Sometimes we want to specify a precise amount of stdout columns, for example in
+// CI or on a cloud environment.
+const columns = process.env.SHOPIFY_CLI_COLUMNS
+if (columns !== undefined && !isNaN(Number(columns))) {
+  process.stdout.columns = Number(columns)
+}
```

Comment:
```suggestion
const columns = Number(process.env.SHOPIFY_CLI_COLUMNS)
if (!isNaN(columns)) {
  process.stdout.columns = columns
}
```

`Number` will return `NaN` if we pass something undefined to it